src/nvim/mark.c

```diff
@@ -89,6 +91,24 @@ void clear_fmark(fmark_T *const fm, const Timestamp timestamp)
   fm->timestamp = timestamp;
 }
 
+/// Execute "MarkSet" autocommand.
+///
+/// @param c The name of the mark, e.g., 'a'.
+/// @param pos Position of the mark in the buffer.
+/// @param buf The buffer of the mark.
+///
+/// @return true if the autocommand was executed.
+static bool apply_markset_autocmd(char c, pos_T *pos, buf_T *buf)
```

Comment:
for better or worse, the common convention is `do_autocmd_`
```suggestion
static bool do_autocmd_markset(char c, pos_T *pos, buf_T *buf)
```